tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -15,6 +15,24 @@
       "IncludeOnlyTargetFrameworks": ["net462"]
     }]
   },
+  {
+    "IntegrationName": "AwsSns",
+    "SampleProjectName": "Samples.AWS.SQS",
+    "NugetPackageSearchName": "AWSSDK.SimpleNotificationService",
+    "MinVersion": "3.0.0",
+    "MaxVersionExclusive": "3.8.0",
+    "SpecificVersions": [
+      "3.1.*",
+      "3.3.*",
+      "3.*.*"
+    ],
+    "VersionConditions": [
+      {
+        "MaxVersionExclusive": "3.3.0",
+        "IncludeOnlyTargetFrameworks": ["net462"]
```

Comment:
There are a large number of changes in this PR around the package versions, and I think it's because your branch is out-of-date with master so running the `GeneratePackageVersions` is creating a lot of different results. Can you rebase onto master and then re-run the build task?

Reply:
I tried rebasing but it was tedious so I just updated master and merged master into my branch and rerunning the build task, it didn't seem to fix much

Reply:
Can you re-run `./build.sh GeneratePackageVersions`? Sorry I should have been more clear that this is the build task that may need to be re-run

Reply:
I reran ./build.sh GeneratePackageVersions

and pretty much got the same changes as before, lots of package versions being edited